### ISSUE TITLE
[Usability][Keyboard Navigation Services->+symbol] Visual focus indicator is cut off for “Privacy statement link”

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1765]
**Actual Result:**
Visual focus indicator is not clearly visible for “Privacy statement link”

**Expected Result:**
Visual focus indicator should be clearly visible for “Privacy statement link”

**Repro Steps:**
1.	Open the Bot Framework V4 Emulator.
2.	Navigate to Welcome tab screen and select it, screen gets opened  
3.	Navigate to the open bot and select it  
4.	Open bot dialog gets opened and connect to the open bot 
5.	Live chat screen gets appears 
6.	Navigate to the Edit ngrok settings and select it 
7.	Emulator screen gets opened 
8.	Navigate to the Privacy statement and observe the issue 

**User Impact:**
Users who rely on Keyboard navigation will be affected the keyboard focus indicator is not clearly visible on control so that keyboard dependant users will be affected.

**Test Environment:**
OS Build: Version 1903(OS Build 18362.239)
Browser: Edge
Application: Bot Framework V4 Emulator
[Usability_settings.zip](https://github.com/microsoft/BotFramework-Emulator/files/3529556/Usability_settings.zip)

**Comment by corinagum** [comment]
Note: focus is cut off. We just need to fix that - no need to involve Design. 
![image](https://user-images.githubusercontent.com/14900841/63546399-97cba400-c4de-11e9-8ed0-4311d558c167.png)

**Comment by tonyanziano** [comment]
Might just be able to throw `width: auto` onto the link and call it a day

**Comment by cwhitten** [comment]
addressed in https://github.com/microsoft/BotFramework-Emulator/pull/1776

**Comment by Kumar2608** [comment]
Verified in latest Bot emulator version -4.5.2-79718 .Issue got fixed hence we are closing this bug.
Now  Visual focus indicator clearly visible for “Privacy statement link”
